core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpDockerCmdExecFactory.java

```diff
@@ -121,7 +121,11 @@ protected Void execute(PingCmd command) {
                 WebTarget webResource = getBaseResource().path("/_ping");
 
                 // TODO contribute to docker-java, make it close the stream
-                IOUtils.closeQuietly(webResource.request().get());
+                try {
+                    webResource.request().get().close();
```

Comment:
I think there may be a subtle bug here; the `get()` call ultimately reaches [this method](https://github.com/testcontainers/testcontainers-java/blob/master/core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java#L269) which has `@SneakyThrows(IOException.class)`. If the get operation fails, `IOException` may be thrown, as if it were an unchecked exception.

It looks like unfortunately the semantics of the new code will catch and ignore the exception from `get()`, instead of letting it bubble up like the old code did.

I could be wrong, but I think this means that the ping command will always succeed, which isn't what we want.